external/llvm-project/mlir/lib/Dialect/Tosa/Transforms/TosaPartition.cpp

```diff
@@ -562,7 +567,7 @@ void TosaPartitionPass::runOnOperation() {
 
       // Grab a useful set of leading ops, like we do for trailing.
       SetVector<Operation *> leadingOps;
-      traceInputs(anchorOp, leadingOps, inputNodes);
+      traceInputs(anchorOp, anchorOp, leadingOps, inputNodes);
```

Comment:
Why anchor op as ignore op here ?

Reply:
Since this routine is now used below for walking up inputs to trailing elementwise ops, the ignoreOp was added so we don't recurse back up the anchor op path. But here it is mostly a placeholder (vs nullptr) since we want to walk back every input of the anchor op.

Reply:
Ack!